docs/shape_batch_size_and_layout.md

```diff
@@ -29,3 +29,12 @@ it ignores the batch_size value.
 on [Shape Inference Document](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_ShapeInference.html).
 In case the model can't be reshaped, it will remain in the original parameters and all requests with incompatible input format
 will get an error. The model server will also report such problem in the logs.
+
+# Changing model input/output layout
```

Comment:
It would be good to include some overview why the layout parameter matters:
OpenVINO models generated via the model optimizer which process image data, have NCHW layout. Image transformation libraries like OpenCV or Pillow use HWC layout. That could make it required to transpose the data on the client before it can be sent to OVMS. Custom nodes implementation as part of DAG scheduler also use CHW format internally to perform image transformations. It could also increase the latency with extra data transposition operations. Layout parameter reduce the latency by changing the model in runtime to accept NHWC layout instead of NCHW. That way the whole processing cycle is more effective thanks to avoided data transpositions. That is especially beneficial for models with high resolution images, where data transposition could be more processing expensive.

Reply:
Added